pkg/crd/generator/testData/config/crds/fun_v1alpha1_toy.yaml

```diff
@@ -133,11 +152,18 @@ spec:
           type: object
         status:
           properties:
+            inter:
+              description: Adding this field for testing interface{} validation
+              oneOf:
+              - type: string
+              - type: integer
+              - type: object
```

Comment:
@DirectXMan12  Do you think `boolean` and `array` type would be valid in this case?

Reply:
According to http://json-schema.org/latest/json-schema-core.html#rfc.section.4.2.1 and if we want this to be exhaustive and tolerated any JSON type, this should include `boolean`, `array`, `null` and `number` as well DYT?